frontend_app_gallery/src/component/Carousel.jsx

```diff
@@ -61,7 +61,7 @@ export class Carousel extends React.Component {
       centerMode: false,
       initialSlide: props.fileSelected,
       swipe: false,
-      arrows: !props.disableAnimation,
+      arrows: !props.disableAnimation && !props.autoPlay,
```

Comment:
FYI: a good practice in variable naming is to try and keep positive naming. It's usually more readable to call you variables props.enableAnimation. Here you are negating a negative variable (!disabled), it's not unclear but it less intuitive.
This comment is for general knowledge, no changes required